src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -609,34 +609,6 @@ public DocumentConventions RegisterAsyncIdConvention<TEntity>(Func<string, TEnti
             return this;
         }
 
-        /// <summary>
-        ///     Register an id convention for a single type (and all its derived types) to be used when calling
-        ///     session.Load{TEntity}(TId id)
-        ///     It is used by the default implementation of FindFullDocumentIdFromNonStringIdentifier.
-        /// </summary>
-        public DocumentConventions RegisterIdLoadConvention<TEntity>(Func<ValueType, string> func)
-        {
-            AssertNotFrozen();
-
-            var type = typeof(TEntity);
-            var entryToRemove = _listOfRegisteredIdLoadConventions.FirstOrDefault(x => x.Item1 == type);
```

Comment:
We need to also remove `_listOfRegisteredIdLoadConventions`, no?

Reply:
I will check this out

Reply:
done